torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -1093,7 +1093,7 @@ PythonArgs PythonArgParser::raw_parse(PyObject* self, PyObject* args, PyObject*
 
 void PythonArgParser::print_error(PyObject* self, PyObject* args, PyObject* kwargs, PyObject* parsed_args[]) {  // NOLINT
   // NOLINTNEXTLINE(clang-analyzer-core.NullDereference)
-  auto num_args = PyTuple_GET_SIZE(args) + (kwargs ? PyDict_Size(kwargs) : 0);
+  auto num_args = static_cast<size_t>(PyTuple_GET_SIZE(args) + (kwargs ? PyDict_Size(kwargs) : 0));
```

Comment:
Is changing the auto with size_t a way to get the same behavior?
Or the static_cast is preferred here?